beard/similarity/pairs.py

```diff
@@ -273,3 +276,124 @@ def transform(self, X):
         X2 = X[:, n_features:]
 
         return np.abs(X1 - X2)
+
+
+class FuzzySimilarity(TransformerMixin):
+
+    """Implements Fuzzy Similarity (Soft TF-IDF) metric."""
```

Comment:
Add a short description of the algorithm. It is not clear otherwise what is this class exactly doing.